### PR TITLE
Update sdk monorepo and storage readme

### PR DESCRIPTION
**Pull request by aayushmahapatra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ yarn add @spheron/storage
 
 In the example below, you can see how to create an instance of `SpheronClient` and how to upload a file/directory to the specified protocol.
 
-```
+```js
```
Comment (shomix) [inline README.md:40]: Nice, didn't know about this.

Should we also do it for the README.md in the `packages/storage/` folder?

**Review by shomix — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (aayushmahapatra) [reply README.md:40]: Done 👍🏻

File: packages/storage/README.md [hunk 2]
```diff
@@ -1,10 +1,33 @@
...
+    <source media="(prefers-color-scheme: light)" srcset="https://github.com/spheronFdn/sdk/blob/main/.github/assets/logo.svg">
+    <img alt="Spheron" src="https://github.com/spheronFdn/sdk/blob/main/.github/assets/logo-dark.svg" width="250">
+  </picture>
+</p>
 
-### Usage:
+<h1 align="center">Spheron Storage SDK</h1>
+
+<p align="center">
+  🧰 Developer toolkits for web3 cloud infrastructure, powered by Spheron.
```
Comment (rekpero) [inline packages/storage/README.md:12]: this should be talking about the multi-chain storage SDK
```suggestion
  🧰 SDK for multi-chain storage, powered by Spheron.
```

**Review by rekpero — CHANGES_REQUESTED** [review]

**Review by rekpero — APPROVED** [review]

File: packages/storage/README.md [hunk 3]
```diff
@@ -1,10 +1,33 @@
...
 
-### Usage:
+<h1 align="center">Storage SDK</h1>
+
+<p align="center">
+  🧰 SDK for multi-chain storage, powered by Spheron.
+</p>
+
+<p align="center">
+  <img src="https://img.shields.io/static/v1?label=npm&message=v14.0.0&color=green" />
```
Comment (rekpero) [inline packages/storage/README.md:16]:
```suggestion
  <img src="https://img.shields.io/static/v1?label=npm&message=v1.0.3&color=green" />
```
This should the current npm package version not node lts

**Review by rekpero — CHANGES_REQUESTED** [review]

File: packages/storage/README.md (hunk 3 above)
Comment (rekpero) [reply packages/storage/README.md:16]: Also this link should redirect user to npm package

File: packages/storage/README.md [hunk 4]
```diff
@@ -1,10 +1,33 @@
...
+  <img src="https://img.shields.io/static/v1?label=license&message=Apache%202.0&color=red" />
```
Comment (rekpero) [inline packages/storage/README.md:17]: this should direct user to github license page

**Review by rekpero — CHANGES_REQUESTED** [review]

File: packages/storage/README.md (hunk 3 above)
Comment (aayushmahapatra) [reply packages/storage/README.md:16]: Github package link or npmjs? 🤔

**Review by rekpero — DISMISSED** [review]

**Review by shomix — APPROVED** [review]

**Review by rekpero — APPROVED** [review]

**Pull request merged**